### ISSUE TITLE
When uploading images, the corresponding image preview cannot be performed

### ISSUE THREAD

**Issue by lzzzzzc** [wkeyuan/DWSurvey#108]
If you need to upload an image, you cannot preview the uploaded image after selecting it, and you cannot determine whether the uploaded image is correct
![C)9I${0HW6GF}G}HK8$O{6U](https://github.com/wkeyuan/DWSurvey/assets/108600719/e6ca29d7-c5d5-443f-aa8b-b78e984f335f)

**Comment by easyandeasy** [comment]
.....

**Comment by Applepie155** [comment]
I have solved the problem that the picture cannot be previewed after uploading, and the specific modification is as follows. You only need to add the corresponding picture preview tag to the answer-p.html file, and it needs to modify the method of uploading file binding. I will give the specific modification method in the form of pictures. If you want to see a preview and release version, I will publish my own version above.
<img width="654" alt="ea56d28effa033791521acb4bc12983" src="https://github.com/wkeyuan/DWSurvey/assets/145668774/585a44ac-a464-49f5-86cf-7b1d961d76b1">
<img width="659" alt="9bcd04f440a94ca05434d8848fe60a0" src="https://github.com/wkeyuan/DWSurvey/assets/145668774/68788793-d98c-449b-a710-da9460cf3cdc">

**Comment by lzzzzzc** [comment]
I have read your code, I think it should be OK, do you have a demonstration of the result diagram, can you show me? Thank you

**Comment by Applepie155** [comment]
Of course, as shown, the preview image is already displayed around here
<img width="668" alt="5f78a7b91659edf6090556b7317f8e7" src="https://github.com/wkeyuan/DWSurvey/assets/145668774/a7749256-d4e2-441f-a84e-0ace7145eb19">
